modules/SecurityGroups/MassAssign.php

```diff
@@ -47,6 +47,13 @@
     } else {
         $where = '';
     }
+    $export_where = !empty($_SESSION['export_where']) ? $_SESSION['export_where'] : '';
+    if (empty($_REQUEST['export_where_md5']) || $_REQUEST['export_where_md5'] != md5($export_where)) {
+        $err = translate('LBL_ERROR_EXPORT_WHERE_CHANGED', 'SecurityGroups');
+        SugarApplication::appendErrorMessage($err);
+        header("Location: index.php?action={$_POST['return_action']}&module={$_POST['return_module']}");
+        sugar_die('');
```

Comment:
Should we be passing in an error message instead of a blank string here?

Reply:
i'm not 100% sure :thinking: